app/resources/add_note_base.html.template

```diff
@@ -196,44 +196,47 @@
         {% endif %}
       </div>
 
-      {% if env.enable_photo_upload %}
-        <div class="photo subsection">
-          <h3>Photo</h3>
-          <div class="hint">
-            {% trans "To attach a photo to this note, upload it or enter its URL." %}
-          </div>
-          <div class="field" onclick="$('note_photo_url_radio').checked=true; update_image_input(true);">
-            <span class="radio">
-                <input id="note_photo_url_radio" type="radio"
-                    checked="checked" name="note_photo_input">
-            </span>
-            <span class="label">
-              {% trans "URL" %}:
-            </span>
-            <span class="value">
-              <input name="note_photo_url" id="note_photo_url"
-                  class="medium-text-input"
-                  value="{{params.photo_url}}" />
-            </span>
-          </div>
-          <div class="field" onclick="$('note_photo_upload_radio').checked=true; update_image_input(true);">
-            {% comment %} This td can be removed if uploading needs to be disabled {% endcomment %}
-            <span class="radio">
-              <input id="note_photo_upload_radio" type="radio"
-                  name="note_photo_input">
-            </span>
-            <span class="label">
-              {% trans "Upload" %}:
-            </span>
-            <span class="value">
-              <input class="photo-upload-input" id="note_photo_upload" type="file"
-                  name="note_photo" disabled="disabled" onchange="show_photo_remove_icon('note_photo_upload')">
-              <a href="javascript:remove_photo_upload('note_photo_upload')" id="remove_note_photo_upload" style="display: none"><img class="photo-upload-cancel" src="{{env.global_url}}/cancel.png" alt="Cancel"></a>
-            </span>
-            {% comment %} see above comment {% endcomment %}
+      {# Hide the photo upload fields for the note record in create form because two photo upload fields show at the same time are a bit confusing. #}
```

Comment:
Optional: It would be better to break this into 2 lines. There is no clear column limit for HTML files, but it should not be too long. Multi line comment should be in format:

```
{# ... #}
{# ... #}
```